package.json

```diff
@@ -41,7 +41,6 @@
   },
   "dependencies": {
     "credit-card-type": "^8.3.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
```

Comment:
this should be dev dependancy

Reply:
nevermind, it turns out I've moved it to dev dependencies and this was an artifact of a deleted branch